resource/subfunctions/custom_field.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Subfunctions
+---
+
+# Custom Field
+
+Resource Administration functions for Custom Field Administration
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _This may not be the only step to manage a **Custom Field** in your environment._
+> _You may also have to refresh the **CFIELD** class to enact these changes._
+> _Please consult RACF documentation and manuals for an understanding of the **CFIELD** class._
+
+## `ResourceAdmin.add_custom_field()`
+
+```python
+  def add_custom_field(
+        self, custom_field_name: str, custom_field_type: str, traits: dict = {}
+  ) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+&nbsp;
+
+{: .experimental }
+> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../../base/segments_traits_operators#segments) and [Traits](../../base/segments_traits_operators#traits) for more details._
+
+&nbsp;
+
+Define a new **Custom Field** profile in the **CFIELD** class.
+
+#### 📥 Parameters
+* `class_name`<br>
+  The name of the **Custom Field** profile being defined to the **CFIELD** class.
+
+* `traits`<br>
+  A dictionary of **traits/attributes** that should be given to the resource on creation. See [Traits](../../base/segments_traits_operators#traits) to see what all of the valid **Resource Traits** are.
+

```

Comment:
Incomplete/incorrect parameter list.

Reply:
Parameter lists for custom field and appc session have been updated.